ffv1.md

```diff
@@ -601,23 +602,25 @@ The end of the bitstream of the `Frame` is filled with 0-bits until that the bit
 This coding mode uses Golomb Rice codes. The VLC is split into two parts. The prefix stores the most significant bits and the suffix stores the k least significant bits or stores the whole number in the ESC case.
 
 ```c
-pseudo-code                                                   | type
---------------------------------------------------------------|-----
-int get_ur_golomb(k) {                                        |
-    for (prefix = 0; prefix < 12; prefix++) {                 |
-        if (get_bits(1)) {                                    |
-            return get_bits(k) + (prefix << k)                |
-        }                                                     |
-    }                                                         |
-    return get_bits(bits) + 11                                |
-}                                                             |
-                                                              |
-int get_sr_golomb(k) {                                        |
-    v = get_ur_golomb(k);                                     |
-    if (v & 1) return - (v >> 1) - 1;                         |
-    else       return   (v >> 1);                             |
+int get_ur_golomb(k) {
+    for (prefix = 0; prefix < 12; prefix++) {
+        if (get_bits(1)) {
+            return get_bits(k) + (prefix << k)
+        }
+    }
+    return get_bits(bits) + 11
+}
+```
+Figure: A pseudo-code description of the read of an unsigned integer in Golomb Rice mode
+
+```c
+int get_sr_golomb(k) {
+    v = get_ur_golomb(k);
+    if (v & 1) return - (v >> 1) - 1;
+    else       return   (v >> 1);
 }
 ```
+Figure: A pseudo-code description of the read of a signed integer in Golomb Rice mode
```

Comment:
Other Figure labels are terminated with `.`, I suggest that here too.

Reply:
> Other Figure labels are terminated with ., I suggest that here too.

Fixed